main_server/local_server/server_admin/c_comdef_admin_main_console.class.php

```diff
@@ -22,9 +22,9 @@
 // #define ( '__NAWS_IMPORT__', 1 )    /* Uncomment to enable the NAWS import functionality. */
 
 /************************************************************************************************************//**
-    \class c_comdef_admin_main_console
-    \brief Controls display of the main BMLT administration console.
-****************************************************************************************************************/
+\class c_comdef_admin_main_console
```

Comment:
Lot of whitespace diffs in this file bro

Reply:
yea wtf super annoying